src/jdk.compiler/share/classes/com/sun/tools/javac/code/Type.java

```diff
@@ -543,6 +543,13 @@ public String toString() {
         if (moreInfo && hasTag(TYPEVAR)) {
             sb.append(hashCode());
         }
+        if (isNullable()) {
```

Comment:
strictly speaking currently we should only have the non-nullable case but I have provided the rest for completeness